tests/system/Storage/RequesterPaysTest.php

```diff
@@ -197,11 +327,13 @@ function (Bucket $bucket, StorageObject $object) {
                 }
             ], [
                 function (Bucket $bucket, StorageObject $object) {
-                    $bucket->iam()->setPolicy([]);
+                    $p = $bucket->iam()->policy();
+                    $bucket->iam()->setPolicy($p);
                 }
             ], [
                 function (Bucket $bucket, StorageObject $object) {
-                    $bucket->iam()->testPermissions(['foo']);
+                    // broke!
```

Comment:
Why is this broke, and should we fix it?

Reply:
sorry, when i was debugging i was having trouble keeping track of provider indexes and noted the broken ones. I missed this when i deleted the other comments. the test works fine.